Formula/dashing.rb

```diff
@@ -12,22 +12,17 @@ class Dashing < Formula
     sha256 "b37d425623bdbb32fe99d58c6d15cbc0753706aad3758aaf95ed229316e2a185" => :el_capitan
   end
 
-  depends_on "glide" => :build
   depends_on "go" => :build
 
   # Use ruby docs just as dummy documentation to test with
   resource "ruby_docs_tarball" do
-    url "https://ruby-doc.org/downloads/ruby_2_5_0_core_rdocs.tgz"
-    sha256 "219e171641e979a5c8dee1b63347a1a26b94ba648aec96f7e6ed915d12bcaa15"
+    url "https://ruby-doc.com/downloads/ruby_2_6_5_core_rdocs.tgz"
```

Comment:
Looks like ruby-doc changed from a `.org` to a `.com`.  I also grabbed the latest version of the docs.